File: C_files/Lecture_1/Pset_1/mario_more/mario.c
#include <stdio.h>

int main()
{
    int height;

    // Get correct input from user
    do
    {
        printf("Height: ");
        scanf("%d", &height);
    }
    while (height < 1);

    // Print the pattern
    for (int i = 0; i < height; i++)
    {
        // Print spaces
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        // Print #
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }

        printf(" ");
        //Print #
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}

File: Python_files/Lecture_6/sentimental-mario-less/mario.py
# TODO
# Get correct input from user
while True:
    try:
        height = int(input('Height: '))
        if height > 0 and height < 9:
            break
    except ValueError:
        print('Invalid input')

# Print the pyramid
for i in range(height):
    for j in range(height - i - 1):
        print(' ', end='')
    for k in range(i + 1):
        print('#', end='')
    print()
    